angular/src/lib/card/examples/card-content.component.ts

```diff
@@ -0,0 +1,62 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'example-card-content',
+  template: `
+    <md-card [ngStyle]="{'width': '300px'}">
```

Comment:
Will a width prop work instead of ngStyle

Reply:
fixed